sp041-20191205T1757.cdl

```diff
@@ -0,0 +1,332 @@
+netcdf sp041-20191205T1757 {
+dimensions:
+	N_SENSOR = 5 ;
+	N_PARAM = 5 ;
+	N_MEASUREMENTS = 25 ;
+variables:
+	string TRAJECTORY ;
+		TRAJECTORY:long_name = "trajectory" ;
+	string SENSOR(N_SENSOR) ;
+		SENSOR:long_name = "name of the sensor" ;
+	string PARAMETER(N_PARAM) ;
+		PARAMETER:long_name = "name of parameter computed from glider measurements" ;
+	string PARAMETER_SENSOR(N_PARAM) ;
+	double TIME_GPS(N_MEASUREMENTS) ;
+		TIME_GPS:_FillValue = -1. ;
+		TIME_GPS:long_name = "time of each gps fix" ;
+		TIME_GPS:units = "seconds since 1970-01-01T00:00:00Z" ;
+		TIME_GPS:ancillary_variables = "TIME_GPS_QC" ;
+	double LATITUDE_GPS(N_MEASUREMENTS) ;
+		LATITUDE_GPS:_FillValue = NaN ;
+		LATITUDE_GPS:long_name = "latitude of each gps fix" ;
+		LATITUDE_GPS:standard_name = "latitude" ;
+		LATITUDE_GPS:units = "degrees_north" ;
+		LATITUDE_GPS:ancillary_variables = "LATITUDE_GPS_QC" ;
+		LATITUDE_GPS:valid_max = "90" ;
+		LATITUDE_GPS:valid_min = "-90" ;
+	double LONGITUDE_GPS(N_MEASUREMENTS) ;
+		LONGITUDE_GPS:_FillValue = NaN ;
+		LONGITUDE_GPS:long_name = "longitude of each gps fix" ;
+		LONGITUDE_GPS:standard_name = "longitude" ;
+		LONGITUDE_GPS:units = "degrees_east" ;
+		LONGITUDE_GPS:ancillary_variables = "LONGITUDE_GPS_QC" ;
+		LONGITUDE_GPS:valid_max = "180" ;
+		LONGITUDE_GPS:valid_min = "-180" ;
+	string PLATFORM ;
+	byte PHASE(N_MEASUREMENTS) ;
+		PHASE:_FillValue = 0b ;
+		PHASE:long_name = "behavior of the glider at sea" ;
+		PHASE:ancillary_variables = "PHASE_QC" ;
+		PHASE:phase_vocabulary = "TBD" ;
+	byte PHASE_QC(N_MEASUREMENTS) ;
+		PHASE_QC:_FillValue = 0b ;
```

Comment:
There is the suggestion: `<PARAM>_QC:_FillValue = " " ;` in the documentation. In this case, since the `PHASE_QC` is a byte type, FillValue should be the same data type. This is a legal and necessary adjust from " " to 0b.